.github/workflows/main.yml

```diff
@@ -140,9 +140,9 @@ jobs:
       run: |
         git clone https://github.com/Microsoft/vcpkg -b 2021.05.12 -c advice.detachedHead=false
         vcpkg/bootstrap-vcpkg.bat -disableMetrics
-        echo $PWD/build/installed/bin | Out-File -FilePath $env:GITHUB_PATH -Append
-        echo $PWD/vcpkg/installed/x64-windows/bin | Out-File -FilePath $env:GITHUB_PATH -Append
-        echo $PWD/vcpkg/installed/x64-windows/tools | Out-File -FilePath $env:GITHUB_PATH -Append
+        Add-Content $env:GITHUB_PATH "$PWD/vcpkg/installed/bin"
```

Comment:
This looks like a really promising change, @kwokcb, and good catch on the shadergen testing!

Could it be that the new build errors are caused by a typo on this line, where `build` has been changed to `vcpkg`?